index.js

```diff
@@ -1,16 +1,67 @@
-var BigNumber = require('bignumber.js')
-
 var Units = {}
 
 var rawUnits = require('./units.json')
 var units = {}
 
 Object.keys(rawUnits).map(function (unit) {
-  units[unit] = new BigNumber(rawUnits[unit], 10)
+  // drop the leading number 1
+  units[unit] = rawUnits[unit].length - 1
 })
 
 Units.units = rawUnits
 
+// add leading zeroes
+// FIXME: improve
+function zeroes (n) {
```

Comment:
> you can preallocate big enough zero string and then just do zeros.slice(0, n)

I was thinking about this, `zeroes` would need to be either 31 zeroes (256bits) or the largest in units.json.

> or you can use https://www.npmjs.com/package/left-pad :)
 
I think I leave left-pad alone :)